core/app.js

```diff
@@ -45,3 +56,34 @@ app.listen(ENV.PORT)
 console.log("Server running on port", ENV.PORT);
 console.log("Development:", !!ENV.HLRDESK_DEV);
 console.log("Hostname:", ENV.HLRDESK_HOST);
+
+
+
+
+
+var sock = require('express')();
+var http = require('http').Server(sock);
+var io = require('socket.io')(http);
+var sock_port = 5431;
+sock.get('/', function(req, res){
+  res.sendFile(__dirname + '/msg.html');
+});
+
+io.on('connection', function(socket){
+  socket.on('write message', function(msg){
+  io.emit('write message', msg);
+});
+});
```

Comment:
Make sure this is indented.

Sorry, I didn't want to say anything but my OCD didn't let me.